src/highlight.rs

```diff
@@ -32,6 +32,11 @@ pub static DATA: LazyLock<Data> = LazyLock::new(|| {
     let dark = Css::new("dark", &DARK_CSS);
     let syntax_set: SyntaxSet =
         syntect::dumps::from_binary(include_bytes!("../assets/newlines.packdump"));
+    let link_highlighting = SyntaxDefinition::load_from_str(
+        include_str!("../assets/LinkHighlight.sublime-syntax"), false, None).expect("loading link style");
+    let mut builder = syntax_set.into_builder();
+    builder.add(link_highlighting);
+    let syntax_set = builder.build();
```

Comment:
This kills debug performance. We could use the trick from https://stackoverflow.com/questions/60751806/how-to-compile-some-dependencies-with-release but I don't know which dependency is the culprit.

Reply:
And it depends on yaml-rust which was promptly denied by CI :(